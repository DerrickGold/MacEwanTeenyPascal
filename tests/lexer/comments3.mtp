(*Regular comment here (*with nested commment here as well
(*maybe third level nested comment*)
new line here*)

finally the last line*)
(*Then the next token is a comment for a challenge *)
end .