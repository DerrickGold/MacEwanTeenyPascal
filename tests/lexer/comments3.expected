7, TOK_KEY_END, end
7, TOK_PERIOD, .
7, TOK_ENDFILE,
